IFC4x3/Properties/h/Hardness_0NzH_YWI19COOLq3fLYBdn/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hardness as a length measure?

Reply:
I think that this is an error on my side. Should probably be IfcPressureMeasure since they specified unit=MPa. Checking with @j-weil just to be certain.